packages/neptune-extension/src/common/components/configure-modal/ConfigureModal.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * todo
+ * - replace base path in auth client
+ * - ui
+ */
+
+
+import React from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {
+  Configuration as BackendApiConfiguration,
+  DefaultApi as BackendApi
+} from 'generated/backend-client/src';
+
+import Modal from 'common/components/modal/Modal';
```

Comment:
If you intend to change or tidy up project organization please factor it out to separate PR. First of all it makes the current commit bloated and second introducing such convention deserves a discussion in my opinion. What about `common/ui/Modal`, which is more domain-oriented ? It seems favourable to group files by functionality, not function signature.

Reply:
I disagree.
We already had convention of grouping `component.tsx` + `component.less` in a folder with the same name. I just fixed that in case of `Modal`.

I don't see benefits of seperate PR, but if you insist I will revert it.

Reply:
My main concern is about the convention itself and pushing it to sth more domain-oriented. Less about whether it will be one or two commits.